src/processout/actionhandler.ts

```diff
@@ -336,26 +357,43 @@ module ProcessOut {
             window.addEventListener("beforeunload", onunload, false);
             window.addEventListener("pagehide", onunload, false); // for iOS and new browsers
 
+            var sightTimer = setInterval(function() {
+                if (!ret.newWindow || ret.newWindow.closed) {
+                    clearInterval(sightTimer);
+                    return;
+                }
+
+                try {
+                    ret.newWindow.postMessage("processout.ping-ready", "*");
+                } catch (err) {
+                    //
```

Comment:
No error catching here?

Reply:
Nope, basically PO.js posts a message to all available iframes to broadcast it everywhere. So, if the merchant has another kind of iframe on the page this could lead to cross origin errors; in these cases we just want to skip them

Reply:
Ok, thanks for the explanation